Content.Client/Administration/UI/Tabs/PlayerTab/PlayerTab.xaml.cs

```diff
@@ -22,28 +22,52 @@ public sealed partial class PlayerTab : Control
         private Header _headerClicked = Header.Username;
         private bool _ascending = true;
 
-        public event Action<BaseButton.ButtonEventArgs>? OnEntryPressed;
+        public event Action<ButtonEventArgs>? OnEntryPressed;
 
         public PlayerTab()
         {
             _adminSystem = EntitySystem.Get<AdminSystem>();
             RobustXamlLoader.Load(this);
             RefreshPlayerList(_adminSystem.PlayerList);
+
             _adminSystem.PlayerListChanged += RefreshPlayerList;
-            OverlayButtonOn.OnPressed += _adminSystem.AdminOverlayOn;
-            OverlayButtonOff.OnPressed += _adminSystem.AdminOverlayOff;
+            _adminSystem.OverlayEnabled += OverlayEnabled;
+            _adminSystem.OverlayDisabled += OverlayDisabled;
+
+            OverlayButton.OnPressed += OverlayButtonPressed;
 
             ListHeader.BackgroundColorPanel.PanelOverride = new StyleBoxFlat(_altColor);
             ListHeader.OnHeaderClicked += HeaderClicked;
         }
 
+        private void OverlayEnabled()
+        {
+            OverlayButton.Pressed = true;
+        }
+
+        private void OverlayDisabled()
+        {
+            OverlayButton.Pressed = false;
+        }
+
+        private void OverlayButtonPressed(ButtonEventArgs args)
+        {
+            if (args.Button.Pressed)
+            {
+                _adminSystem.AdminOverlayOn();
+            }
+            else
+            {
+                _adminSystem.AdminOverlayOff();
+            }
+        }
+
         protected override void Dispose(bool disposing)
         {
             base.Dispose(disposing);
 
             _adminSystem.PlayerListChanged -= RefreshPlayerList;
-            OverlayButtonOn.OnPressed -= _adminSystem.AdminOverlayOn;
-            OverlayButtonOff.OnPressed -= _adminSystem.AdminOverlayOff;
+            OverlayButton.OnPressed -= OverlayButtonPressed;
```

Comment:
Unsubscribe overlay enabled / disabled as well?

Reply:
Fixed in [`5c60cd2` (#7153)](https://github.com/space-wizards/space-station-14/pull/7153/commits/5c60cd2b91e44a0aebeec5809918f29f5656dbfb)